### PR TITLE
tests/bank: add bank case

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

Add bank test case.

### Check List 

Tests 

 - Integration test

Related changes

 - Need to cherry-pick to the release branch

### Release note

- No release note.

### CODE REVIEW

**Comment by amyangfei** [comment]
Please fix CI @overvenus

**Comment by overvenus** [comment]
PTAL

**Comment by amyangfei** [comment]
@JinLingChristopher will help to finish this PR 👏

**Comment by JinLingChristopher** [comment]
This passes the ci but need to revise twice to make it cover more corner scenario

**Comment by JinLingChristopher** [comment]
PTAL @overvenus

**Comment by JinLingChristopher** [comment]
need to fix eventually consistent

**Comment by JinLingChristopher** [comment]
@amyangfei, this pr is ready to be reviewd.

**Comment by overvenus** [comment]
LGTM, thank you! @JinLingChristopher

**Review by liuzix — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

File: tests/bank/run.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+cd "$(dirname "$0")"
+set -euxo pipefail
+
+GO111MODULE=on go run bank.go case.go -u "root@tcp(${UP_TIDB_HOST}:${UP_TIDB_PORT})/bank" \
+    -d "root@tcp(${DOWN_TIDB_HOST}:${DOWN_TIDB_PORT})/bank" \
+    --test-round=20
+
+cleanup_process $CDC_BINARY
+echo "[$(date)] <<<<<< run test case $TEST_NAME success! >>>>>>"
```
Comment (amyangfei) [inline tests/bank/run.sh:44]: The kafka test failed, because this case only skips `sink-type=kafka` in the `prepare` function.

**Comment by amyangfei** [comment]
Good job @JinLingChristopher

**Pull request merged**